docs/iris/src/developers_guide/release.rst

```diff
@@ -0,0 +1,66 @@
+Release
+*******
+
+A release of Iris is a tag on github in the SciTools organisation master.
+
+
+Release Branch
+==============
+
+Once the features intended for the release are on master, a release branch should be created, in the SciTools/iris repository.  This will have the name:
+
+    :literal:`{major release number}.x`
+
+for example:
+
+    :literal:`v1.9.x`
+
+This branch shall be used to finalise the release details in preparation for the release candidate.
+
+Release Candidate
+=================
+
+Prior to a release, a release candidate tag shall be created, marked as a pre-release in github, with a tag ending with :literal:`rc` followed by a number, e.g.:
+
+    :literal:`v1.9.0rc1`
```

Comment:
While compatible with SemVer, I don't think that string would be compatible with PEP440. See third paragraph in [this section of the spec](https://www.python.org/dev/peps/pep-0440/#semantic-versioning) which states

> Semantic versions containing a hyphen (pre-releases - clause 10) or a plus sign (builds - clause 11) are not compatible with this PEP and are not permitted in the public version field.

Note these clauses reference SemVer 2.0.0-rc-1 and are one less in the final SemVer 2.0.0.